packages/cms/src/components/editors/SimpleVisualizationEditor.jsx

```diff
@@ -60,6 +60,18 @@ class SimpleVisualizationEditor extends Component {
     }
   }
 
+  extractPayload(resp) {
+    if (resp.data.data) {
+      return resp.data.data;
+    }
+    else if (resp.data) {
+      return resp.data;
+    }
+    else {
+      return resp;
+    }
+  }
+
```

Comment:
Since `resp` is always coming from axios, then we will ALWAYS need to do `resp.data` (because that's the actual payload). This if/else tree check for 3 things:
1. check for `resp.data` and `resp.headers` like d3plus-viz: https://github.com/d3plus/d3plus-viz/blob/master/src/data/load.js#L79
2. check for `resp.data`
3. return `resp`